Minor cleanup of fzf-tmux

a1346054 opened the pull request.

junegunn commented on bin/fzf-tmux:
```diff
@@ -7,8 +7,8 @@ fail() {
 
-fzf="$(command -v fzf 2> /dev/null)" || fzf="$(dirname "$0")/fzf"
-[[ -x "$fzf" ]] || fail 'fzf executable not found'
+fzf=$(command -v fzf 2>/dev/null) || fzf="$(dirname "$0")/fzf"
```
Why do we remove the space before `/dev/null`?

junegunn replied:
Related? https://github.com/koalaman/shellcheck/issues/1344#issuecomment-443542106

a1346054 replied:
Was just a stylistic choice, I removed it now.

a1346054 commented:
I rebased the pull request on latest master to fix merge conflicts, and reworked the commits to make more minimal changes, avoiding stylistic fixes.

It makes the scripts more robust, and tiny bit faster because bash doesn't have to start up the globbing logic when variables are properly quoted.
